webex_assistant_sdk/crypto.py

```diff
@@ -41,7 +40,7 @@ def verify_signature(secret: str, message: bytes, signature: bytes) -> None:
 def load_private_key(private_key_bytes: bytes):
     """Loads a private key in PEM format"""
     try:
-        private_key: RSAPrivateKey = load_ssh_private_key(private_key_bytes, None)
+        private_key: RSAPrivateKey = load_pem_private_key(private_key_bytes, None)
```

Comment:
Could you update the methods here:
- https://github.com/cisco/webex-assistant-sdk/blob/master/get_started_documentation/echo-skill-secure/echo_skill_secure_tester/main.py#L17
- https://github.com/cisco/webex-assistant-sdk/blob/master/get_started_documentation/echo-skill-secure/echo_skill_secure/main.py#L21

So that they reflect the new encryption logic.

Reply:
Good catch. Fixed.